packages/gcf-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcf-utils",
-  "version": "16.2.1",
+  "version": "16.2.2",
```

Comment:
Release Please maintains this line. I think this manual edit will be overridden by it (with the same value).

![image](https://github.com/user-attachments/assets/8dd0ef3d-b16b-45f3-a1eb-d4950af71c73)